backend.py

```diff
@@ -62,18 +62,18 @@ def walk(self, distance, state, direction=None, push_others=True):
                 # Check the absence of a walls before moving.
                 if not check_the_absence_of_a_wall(self.coordinates, direction, state):
                     break
-                    
+
                 # There is no wall. Get next coordinates.
                 next_coordinates = get_next_coordinates(self.coordinates, direction)
                 # Check robots on the next tile before moving.
-                robot_in_the_way_index = check_robot_in_the_way(state, next_coordinates)
+                robot_in_the_way = check_robot_in_the_way(state, next_coordinates)
 
                 # Move robot in the way.
-                if robot_in_the_way_index is not None:
+                if robot_in_the_way is not None:
```

Comment:
```suggestion
                if not robot_in_the_way:
```
(Kdyz vracime robota, nikoli index, jde to napsat i takto - citelneji)